src/Microsoft.AspNetCore.SystemWebAdapters/Hosting/HostingEnvironment.cs

```diff
@@ -1,56 +1,56 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What changed in this file that made every line but 1 show a change with no actual changes?